src/mongo/JobGroup.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Tripod\Mongo;
```

Comment:
I had thought of this, and I don't *really* care one way or the other.  The reason I didn't put it in `jerbs` is because it's not actually a job.  It's really more of a model, but we don't have any analog for that in tripod.